src/components/UserProfile.vue

```diff
@@ -59,15 +68,20 @@ export default {
   methods: {
     updateUser(evt) {
       evt.preventDefault();
-      this.$apollo.mutate({
-        mutation: UPDATE_USER_PROFILE,
-        variables: {
-          user: {
-            email: userEmail.value,
-            receiveEmails: this.receiveEmails
+      this.$apollo
+        .mutate({
+          mutation: UPDATE_USER_PROFILE,
+          variables: {
+            user: {
+              email: this.currentUser.email,
```

Comment:
Ended up using the variable defined within the model. This seemed to get rid of the lint errors and fix some weirdness